File: 88-merge-sorted-array/88-merge-sorted-array.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
    
        int index = 0, min, temp;
        for(int i=m; i<m+n; i++){
            nums1[i] = nums2[index];
            index++;
        }
        for(int i=0; i<m+n-1; i++){
            min = nums1[i];
            index = i;
            for(int j=i+1; j<m+n; j++){
                if(nums1[j] < min){
                    index = j;
                    min = nums1[j];
                }
            }
            temp = nums1[i];
            nums1[i] = nums1[index];
            nums1[index] = temp;
        }
    }
}

File: 88-merge-sorted-array/88-merge-sorted-array.c


void merge(int* nums1, int nums1Size, int m, int* nums2, int nums2Size, int n){
    int index = 0, temp;
    for(int i=m; i<m+n; i++){
        nums1[i] = nums2[index];
        index ++;
    }
    for(int i=1; i<m+n; i++){
        for(int j=i; j>0; j--){
            if(nums1[j] < nums1[j-1]){
                temp = nums1[j-1];
                nums1[j-1] = nums1[j];
                nums1[j] = temp;
            }
        }
    }
}